core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/machineLearning/Scorer/ScorerOpDesc.scala

```diff
@@ -0,0 +1,189 @@
+package edu.uci.ics.texera.workflow.operators.machineLearning.Scorer
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{
+  JsonSchemaInject,
+  JsonSchemaString,
+  JsonSchemaTitle
+}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.{
+  AutofillAttributeName,
+  HideAnnotation
+}
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+
+class ScorerOpDesc extends PythonOperatorDescriptor {
+  @JsonProperty(required = false, defaultValue = "false")
+  @JsonSchemaTitle("Regression")
+  @JsonPropertyDescription(
+    "Choose to solve a regression task"
+  )
+  var isRegression: Boolean = false
+
+  @JsonProperty(required = true)
+  @JsonSchemaTitle("Actual Value")
+  @JsonPropertyDescription("Specify the label column")
+  @AutofillAttributeName
+  var actualValueColumn: String = ""
+
+  @JsonProperty(required = true)
+  @JsonSchemaTitle("Predicted Value")
+  @JsonPropertyDescription("Specify the attribute generated by the model")
+  @AutofillAttributeName
+  var predictValueColumn: String = ""
+
+  @JsonProperty(required = false, value = "classificationFlag")
+  @JsonSchemaTitle("Scorer Functions")
+  @JsonPropertyDescription("Select classification tasks metrics")
+  @JsonSchemaInject(
+    strings = Array(
+      new JsonSchemaString(path = HideAnnotation.hideTarget, value = "isRegression"),
+      new JsonSchemaString(path = HideAnnotation.hideType, value = HideAnnotation.Type.equals),
+      new JsonSchemaString(path = HideAnnotation.hideExpectedValue, value = "true")
+    )
+  )
+  var classificationMetrics: List[classificationMetricsFnc] = List()
+
+  @JsonProperty(required = false, value = "regressionFlag")
+  @JsonSchemaTitle("Scorer Functions")
+  @JsonPropertyDescription("Select regression tasks metrics")
+  @JsonSchemaInject(
+    strings = Array(
+      new JsonSchemaString(path = HideAnnotation.hideTarget, value = "isRegression"),
+      new JsonSchemaString(path = HideAnnotation.hideType, value = HideAnnotation.Type.equals),
+      new JsonSchemaString(path = HideAnnotation.hideExpectedValue, value = "false")
+    )
+  )
+  var regressionMetrics: List[regressionMetricsFnc] = List()
+
+  override def operatorInfo: OperatorInfo =
+    OperatorInfo(
+      "Scorer",
+      "Scorer for machine learning models",
+      OperatorGroupConstants.MODEL_PERFORMANCE_GROUP,
+      inputPorts = List(InputPort()),
+      outputPorts = List(OutputPort())
+    )
+  override def getOutputSchema(schemas: Array[Schema]): Schema = {
+    val outputSchemaBuilder = Schema.builder()
+    outputSchemaBuilder.add(new Attribute("Label", AttributeType.STRING))
+
```

Comment:
I understand you may need output metrics for each class for classifiers, and that's why you add `Label` attribute. but why do you need `Label` attribute for regression? 

I suggest just name it `Class` attribute for classifier only. for regression, we can omit this.

Reply:
Originally, our ML workflow can do the hyperparameter tuning so it will generate multiple models. Therefore, we introduce another operator "ModelSelection" to choose the best performance model based on Overall (the value in Label). I am not sure whether this function merge into the master branch in the future so I keep it.

Reply:
But, I can remove this column right now.

Reply:
fix